app/models/transformation_mapping.rb

```diff
@@ -1,9 +1,53 @@
 class TransformationMapping < ApplicationRecord
+  VM_VALID = N_("OK").freeze
```

Comment:
You do not want to set this as a constant.  It will get set once with whatever localization is in place when the model loads.  You are trying to honor the current users local which is dynamic.